HannaPidchasiuk/HW8/task2.py

```diff
@@ -0,0 +1,17 @@
+# Implement your realization of the function filter
+def my_filter(callback, sequence) -> list:
+    return [item for item in sequence if callback(item)]
+
+
+def check_is_true(item):
```

Comment:
add anotations

Reply:
or even better implement lambda function to pactice

Reply:
fixed